app/views/works/homepage.html.erb

```diff
@@ -0,0 +1,39 @@
+<% if @works.nil? == false %>
+  <h1>This week's spotlight : <%= @works.spotlight.title %> By: <%= @works.spotlight.creator %></h1>
+<% end %>
+
+<div class="col-1">
+  <h1>Top Books</h1>
+  <ul>
+    <% @works.top10("book").each do |work| %>
+      <li>
+        <%= link_to work.title, work_path(id: work[:id]), class: "link" %>
+        <p><%= work.votes.count %> votes</p>
+      </li>
+    <% end %>
+  </ul>
```

Comment:
Did you notice you do a lot of these lists?  You can Dry up this code by making a partial and passing it in a collection of works to render.